crates/utils/src/error.rs

```diff
@@ -212,6 +212,7 @@ pub enum LemmyErrorType {
   CouldntSendWebmention,
   ContradictingFilters,
   InstanceBlockAlreadyExists,
+  #[serde(rename = "`jwt` cookie must be marked secure and httponly")]
```

Comment:
same as the other PR: i don't think using "rename" is the correct way to add messages to this enum. (see https://github.com/LemmyNet/lemmy/pull/4007#discussion_r1345783918 )

Reply:
Changed this to a comment instead.